CIPs/CIP-85/CIP-85.md

```diff
@@ -86,80 +82,86 @@ The `CollectionSlice` schema must be an `object` with a `$ceramic` field from [C
   title: 'MyCollectionSLice',
   type: 'object',
   properties: {
+    collection: { type: 'string', maxLength: 150 },
```

Comment:
Don't we have a new special way to refer to ceramic DocIDs in json-schemas?

Reply:
Yes we could add the `$ceramic` field here to be more explicit, but we already have `$ceramic: { type: 'collectionSlice', schema: '<slice schema ID>' }` on the object itself so I would think this is enough information for the tools to act on, knowing what properties are contained by the `collectionSlice`?